Utils/VersionBump/main.swift

```diff
@@ -62,3 +71,50 @@ func extractVersionOnly(_ version: String) -> String {
     guard let indexOfHypen = version.firstIndex(of: "-") else { return version }
     return String(version.prefix(upTo: indexOfHypen))
 }
+
+func verifyVersionInFiles() throws {
+    let expectedVersion = args[2]
+    
+    for file in files {
+        let fileHandler = try open(file)
+        let fileContent: String = fileHandler.read()
+
+        var regexString: String = ""
+        if file.hasSuffix(".podspec") {
+            if file == "./Tests/HybridSDKTest/HybridPod.podspec" {
+                regexString = "s\\.dependency\\s\"Sentry\\/HybridSDK\",\\s\"(?<version>[a-zA-z0-9\\.\\-]+)\""
+            } else {
+                regexString = "\\ss\\.version\\s+=\\s\"(?<version>[a-zA-z0-9\\.\\-]+)\""
+            }
+        } else if file == "./Package.swift" {
+            regexString = "https:\\/\\/github\\.com\\/getsentry\\/sentry-cocoa\\/releases\\/download\\/(?<version>[a-zA-z0-9\\.\\-]+)\\/Sentry"
+        } else if file == "./Sources/Sentry/SentryMeta.m" {
+            regexString = "static NSString \\*versionString = @\"(?<version>[a-zA-z0-9\\.\\-]+)\""
+        }
+        let match = try? Regex(string: regexString, options: [.dotMatchesLineSeparators]).firstMatch(in: fileContent)
+        let version = match?.captures[0] ?? "Version Not Found"
+        if version != expectedVersion {
+            exit(errormessage: "Unexpected version \(version) found for file '\(file)'")
```

Comment:
I would either fail faster on the `try?` by converting it to `try` or, my preference, gather all the failures in the for loop and then print a summary at the end. It could be helpful to know if only a subset of the versions weren't updated correctly, vs failing fast and then having to push multiple fixes until all problems are solved.

That would indicate we should change the `try` on line 79 to a `try?` as well, or catch the error and store for later while trying the rest of the files.